sdk/core/azure-core/src/main/java/com/azure/core/util/polling/DefaultPollingStrategy.java

```diff
@@ -76,9 +76,27 @@ public DefaultPollingStrategy(HttpPipeline httpPipeline, JsonSerializer serializ
      * @throws NullPointerException If {@code httpPipeline} is null.
      */
     public DefaultPollingStrategy(HttpPipeline httpPipeline, String endpoint, JsonSerializer serializer, Context context) {
+        this(httpPipeline, endpoint, serializer, null, context);
+    }
+
+    /**
+     * Creates a chained polling strategy with 3 known polling strategies, {@link OperationResourcePollingStrategy},
+     * {@link LocationPollingStrategy}, and {@link StatusCheckPollingStrategy}, in this order, with a custom
+     * serializer.
+     *
+     * @param httpPipeline an instance of {@link HttpPipeline} to send requests with.
+     * @param endpoint an endpoint for creating an absolute path when the path itself is relative.
+     * @param serializer a custom serializer for serializing and deserializing polling responses.
+     * @param serviceVersion the service version that will be added as query param to each polling
+     * request and final result request URL. If the request URL already contains a service version, it will be replaced
+     * by the service version set in this constructor.
+     * @param context an instance of {@link Context}.
+     * @throws NullPointerException If {@code httpPipeline} is null.
+     */
+    public DefaultPollingStrategy(HttpPipeline httpPipeline, String endpoint, JsonSerializer serializer, String serviceVersion, Context context) {
```

Comment:
Do we want make a properties bag for the constructor? There have been a lot of constructors added already.